cylc/flow/network/scan.py

```diff
@@ -164,8 +165,10 @@ def scan_many(items, methods=None, timeout=None, ordered=False):
         try:
             msg = await client.async_request(method)
         except ClientTimeout as exc:
+            LOG.exception("ClientTimeout")
```

Comment:
Can we add the reg, host and port to the error message to help identify the server to which the issue refers.

Reply:
Not strictly necessary as that info is returned next and printed to stdout anyway.  But fair enough, better to print it with the exception as well.

Reply:
Due to the async implementation I don't think we can rely on the stdout context any more.